client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -179,9 +179,7 @@ class SharingServiceDescription extends Component {
 			'google_photos' === this.props.service.ID &&
 			'must-disconnect' === this.props.status
 		) {
-			description = this.props.translate(
-				'Please connect again to continue using Photos for Google.'
-			);
+			description = this.props.translate( 'Please connect again to continue using Google Photos.' );
```

Comment:
:disappointed: 19 existing translations will be lost with this change.

Reply:
I don't think so

Reply:
@johngodley why not? I mean it's still a required change to make and we'll make it, but I couldn't found the new string anywhere.

Reply:
I'm a little confused. The bot said 19 translations will be lost, which I assumed meant that 19 strings have been changed, when only 1 has.

On re-reading I think it might mean that 19 languages have this translated, and those will be lost?

Reply:
Correct. Again, this is perfectly normal with this type of change. We added this kind of notification just in case some changes are not really necessary, or even sometimes unintentional (not the case here)